f5/bigip/resource.py

```diff
@@ -742,9 +745,11 @@ def _load(self, **kwargs):
         base_uri = self._meta_data['container']._meta_data['uri']
         kwargs.update(requests_params)
         response = refresh_session.get(base_uri, **kwargs)
-        self._local_update(response.json())
-        self._activate_URI(self.selfLink)
-        return self
+        # Make new instance of self
```

Comment:
Hang on this code surprises me!   Why isn't `create` just calling `_create`?  That's the standard pattern...   isn't it?    I might have to `git blame` this code.